nativescript-imagepicker/albums.ios.js

```diff
@@ -40,3 +49,34 @@ function done(args) {
     page.bindingContext.done();
 }
 exports.done = done;
+function albumsPageFactory() {
```

Comment:
I hoped we will be able to provide proper WebPack config that will copy the XML.